operator/pkg/manifest/installer.go

```diff
@@ -376,6 +381,9 @@ func ApplyManifest(componentName name.ComponentName, manifestStr, version string
 		o.AddLabels(map[string]string{istioComponentLabelStr: string(componentName)})
 		o.AddLabels(map[string]string{operatorLabelStr: operatorReconcileStr})
 		o.AddLabels(map[string]string{istioVersionLabelStr: version})
+		if revision != "" {
+			o.AddLabels(map[string]string{model.RevisionLabel: revision})
+		}
```

Comment:
could we consolidate the add labels logic with the selection labels above? i think they could be identical. not an issue with your pr's additions but seems like a good opportunity to clean this up a bit.

Reply:
Sure

Reply:
Actually no, I don't really know what you mean. We need this to happen here and the selection to happen previously since the add/delete path is different

Reply:
per offline discussion, ignore my comment.